scripts/EN/Cybil

```diff
@@ -1786,21 +1786,21 @@ DrawBackground(); // 0x23c8: 00000065
 PlaySoundFx(0x39, 0x1); // 0x23cc: 00002072
 ShowText((7080)"It was a slightly dirty, but familiar doll."); // 0x23d4: 0000100c
 HandleInput(); // 0x23da: 00000010
-ShowText((7081)"Countless blue fireworks went off in front of Cybil's eyes. Memories of the past... Unforgettable memories... Memories that should have been tightly sealed away within her..."); // 0x23de: 0000100c
+ShowText((7081)"Countless blue sparks danced in front of Cybil's eyes.¥n¥nW2Memories of the past…¥n¥nW2Unforgettable memories…¥n¥nW2Memories that should have been tightly sealed away within her…"); // 0x23de: 0000100c
```

Comment:
Changed fireworks to sparks and used danced instead of the more literal scattered. We could use stars instead of sparks too...